source/SIL.AppBuilder.Portal.Frontend/src/data/query.tsx

```diff
@@ -93,12 +93,8 @@ export function queryApi<T>(mapRecordsToProps, options?: IQueryOptions) {
           return queryResult;
         });
 
-        try {
-          await Promise.all(requestPromises);
```

Comment:
the main issue is that after a 500 error, all promises seem to never resolve

Reply:
this was because the requestQueue is halted on a 500 to give us a chance to handle the error if we need to